hazelcast-client-new/src/test/java/com/hazelcast/client/spi/impl/discovery/ClientDiscoverySpiTest.java

```diff
@@ -179,10 +208,36 @@ public void testNodeFilter_from_xml() throws Exception {
         assertEquals(4, nodeFilter.getNodes().size());
     }
 
+    @Test
+    public void test_discovery_address_translator() throws Exception {
+        String xmlFileName = "hazelcast-client-discovery-spi-test.xml";
+        InputStream xmlResource = ClientDiscoverySpiTest.class.getClassLoader().getResourceAsStream(xmlFileName);
+        ClientConfig clientConfig = new XmlClientConfigBuilder(xmlResource).build();
+
+        ClientNetworkConfig networkConfig = clientConfig.getNetworkConfig();
+
+        DiscoveryStrategiesConfig discoveryStrategiesConfig = networkConfig.getDiscoveryStrategiesConfig();
+        assertNotNull(discoveryStrategiesConfig);
+        assertNotNull(discoveryStrategiesConfig.getNodeFilterClass());
+        ILogger logger = Logger.getLogger(ClientDiscoverySpiTest.class);
```

Comment:
I would like to do this but wasn't that the problem with configuration since the loggers are not configured from the xml (like loggertype set to log4j?)